Source/BaaService.swift

```diff
@@ -76,6 +85,53 @@ public extension BaaService {
     func delete(forHashValue hashValue: String) throws {
```

Comment:
I see we have `save` and `delete`. I think we should consider `update` method. But its not needed now, it could be an improvement in the future.